### PR TITLE
ENG-19977 expose subAccountAppId

### PR DESCRIPTION
**Pull request by yangAtSpoken**

### CODE REVIEW

File: NodeJS/README.md [hunk 1]
```diff
@@ -46,6 +46,9 @@ The API provides AgentClient class that could be used to create and delete agent
     // retrieve token for agent
     let token = agentClient.getUserToken(agentUsername)
 
+    // get sub account app id
+    let subAccountAppId = agentClient.SubAccountAppId
```
Comment (joe-s-avaya) [inline NodeJS/README.md]: nit: should use `const` unless the value is expected to change 
```suggestion
    const subAccountAppId = agentClient.SubAccountAppId
```

File: NodeJS/sample/SampleAgentClient.ts [hunk 2]
```diff
@@ -55,6 +55,7 @@ async function deleteAgent(agentClient: AgentClient) {
 
 async function main() {
   const agentClient = await createAgentClient(endpoint, apiKey)
+  console.log('subaccount app id is ', agentClient.SubAccountAppId)
```
Comment (joe-s-avaya) [inline NodeJS/sample/SampleAgentClient.ts]: I would strongly recommend NOT checking-in `console.log` statements, even if this is just a sample app. I won't block on it, but it is a code-smell to me.

File: NodeJS/src/RestClient.ts [hunk 3]
```diff
@@ -130,7 +129,7 @@ export class RestClient {
     const id = await this.getSubAccount().then((response: { id: any }) => {
       return response.id
     })
-    console.log(`id is ${id}`)
+    logger.debug(`id is ${id}`)
```
Comment (joe-s-avaya) [inline NodeJS/src/RestClient.ts]: nit: for clarity, I'd recommend being more explicit on _what_ "id" this is
```suggestion
    logger.debug(`Sub Account App ID is: ${id}`)
```

File: NodeJS/__unit__/AgentClient.test.ts [hunk 4]
```diff
@@ -9,6 +9,9 @@ describe('AgentClient.ts', () => {
     const restClient: any = RestClient as jest.Mock
     client = new AgentClient('1', 'MYA_MYARec', restClient)
   })
+  test('get SubAccountAppId should return subAccountAppId', async () => {
+    expect(client.SubAccountAppId).toBe('MYA_MYARec')
```
Comment (joe-s-avaya) [inline NodeJS/__unit__/AgentClient.test.ts:13]: very happy to see you adding unit tests for even the simple things! 💯

**Review by joe-s-avaya — APPROVED** [review]
💯  nice

**Review by ilovett — APPROVED** [review]

**Pull request merged**